Mergin/switch_sources_dialog.py

```diff
@@ -0,0 +1,202 @@
+import os
+import yaml
+from dataclasses import dataclass, field
+import typing
+import re
+
+import psycopg2
```

Comment:
Hmm I am not sure if all QGIS installs will have psycopg2 automatically installed. May be good to handle importerror here, just to make sure that missing psycopg will not make the whole plugin unusable...